app/models/mentor/student.rb

```diff
@@ -6,12 +6,15 @@ class Student
       '_application_coding_level',
       '_application_code_samples',
       '_application_learning_history',
-      '_application_skills',
-      '_application_code_background'
+      '_application_language_learning_period',
+      '_application_skills'
+      
     ]
 
     attr_accessor :coding_level, :code_samples
-    attr_accessor :learning_history, :code_background
```

Comment:
`code_background` tends to get a little on the personal side, and is often just a lot of information that is not super necessary to mentors. I think we were mainly worried here about how much time mentors would spend reading this info if it's not particularly relevant to the skills they would bring to the table.